src/components/VerticalStepper/VerticalStepper.tsx

```diff
@@ -24,7 +27,7 @@ export const VerticalStepper = ({ state, activeStep, steps }: VerticalStepperPro
         size='sm'
       >
         {steps.map(step => (
-          <Step key={step.label} label={step.label} />
+          <Step key={step.label} label={step.label} isCompletedStep={step.isCompleted} />
```

Comment:
Nitpick: Could just be `isCompleted`, the component is called `<Step />` so we're obviously talking about a step.